x-pack/plugin/rank-rrf/src/yamlRestTest/resources/rest-api-spec/test/rrf/150_rank_rrf_pagination.yml

```diff
@@ -0,0 +1,1055 @@
+setup:
+  - skip:
+      version: ' - 8.14.99'
+      reason: 'pagination for rrf was added in 8.15'
```

Comment:
You should adjust this. Also, once this is merged, you will need to adjust main to make sure the older versions are tested.

Reply:
updated in 59fd2cdbf42f96aa2e89237a4cc73e5856d75a69